java/core/src/main/java/sleeper/core/statestore/FileReferenceCount.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.core.statestore;
+
+import java.util.Objects;
+
+public class FileReferenceCount {
+    private final Long lastUpdateTime;
+    private final String tableId;
```

Comment:
The tableId isn't needed here, as the state store exists in a context where the table is already known.